ui/ui-expenses/src/screens/list/list.utils.js

```diff
@@ -0,0 +1,45 @@
+
+const addImageProcess = (src) => {
+  return new Promise((resolve, reject) => {
+    // Create an abstract canvas and get context
+    const mycanvas = document.createElement("canvas");
```

Comment:
this is similar to the process done with a classical upload.
but you don't have the resize here.
try to reuse the classical process (with resizing)